jupyterhub/Chart.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v2
 name: jupyterhub
 version: 0.0.1-set.by.chartpress
-appVersion: "2.3.1"
+appVersion: "3.0.0-beta.1"
```

Comment:
Does helm care that appVersion is strict go-semver format, or is a normalized Python version string accepted?